incubator/fluentd/values.yaml

```diff
@@ -9,11 +9,12 @@ image:
   #   - secret1
   #   - secret2
 
-output:
-  host: elasticsearch-client.default.svc.cluster.local
-  port: 9200
-  buffer_chunk_limit: 2M
-  buffer_queue_limit: 8
+env:
+  OUTPUT_HOST: elasticsearch-client.default.svc.cluster.local
+  OUTPUT_PORT: 9200
+  OUTPUT_BUFFER_PATH: /var/log/fluentd-buffers
+  OUTPUT_BUFFER_CHUNK_LIMIT: 2M
+  OUTPUT_BUFFER_QUEUE_LIMIT: 8
```

Comment:
This change makes it backward incompatible! Current users has to update their values files before using the new version. So we either have to bump the minor field in version rather than the patch field, or keep the old values structure and just add new fields with default values that wouldn't change how it works with existing values files.

Reply:
I'll keep the old values structure to make it backwards compatible